iep-lwcapi/src/main/scala/com/netflix/iep/lwcapi/EddaObservable.scala

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2014-2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.iep.lwcapi
+
+import java.io.IOException
+
+import com.fasterxml.jackson.databind.JsonMappingException
+import com.netflix.iep.json.Json
+import com.typesafe.scalalogging.StrictLogging
+import rx.lang.scala.Observable
+
+import scala.util.control.NonFatal
+import scalaj.http._
+
+//
+// Return a set of instances we should try connecting to.
+// As we do not send to these nodes, we only collect data from them,
+// it's important we connect before they are accepting client traffic
+// so we don't drop data.
+//
```

Comment:
Can you follow the typical java/scala comment format so these will show up in scaladoc or similar tools?

```
/**
  * ...
  */
```